geomesa-arrow/geomesa-arrow-datastore/src/main/scala/org/locationtech/geomesa/arrow/data/ArrowFeatureStore.scala

```diff
@@ -61,10 +61,9 @@ class ArrowFeatureStore(entry: ContentEntry, reader: SimpleFeatureArrowFileReade
     require((flags | WRITER_ADD) == WRITER_ADD, "Only append supported")
 
     val sft = delegate.getSchema
-    val os = entry.getDataStore.asInstanceOf[ArrowDataStore].createOutputStream(true)
+    val os = entry.getDataStore.asInstanceOf[ArrowDataStore].createOutputStream() // append = true
```

Comment:
Do we need the comment here?  (It may be sensible.  Just checking)

Reply:
well intellij complained that i was calling with a default argument, so i removed it... but i kind of prefer it to be explicit so i put the comment